pwndbg/commands/config.py

```diff
@@ -50,15 +52,25 @@ def config():
                        '- then put it in your .gdbinit after initializing pwndbg'))
 
 
-@pwndbg.commands.Command
-def configfile(show_all=False):
-    """Generates a configuration file for the current Pwndbg options"""
+parser = argparse.ArgumentParser(
+    description='Generates a configuration file for the current Pwndbg options'
+)
+parser.add_argument('show_all', nargs='?', default=False, type=bool, help='Whether to show all config params')
```

Comment:
Might be worth changing this to `--all` since it's really an optional flag (versus a positional)